addons/website/views/snippets/s_kickoff.xml

```diff
@@ -8,7 +8,7 @@
         </span>
         <div class="o_we_bg_filter bg-black-50"/>
         <div class="o_we_shape o_html_builder_Connections_06" style="background-image: url('/html_editor/shape/html_builder/Connections/06.svg?c5=o-color-4');"/>
-        <div class="container">
+        <div class="container s_allow_columns" data-vxml="001">
```

Comment:
As said in another comment, the `data-vxml` should go on the snippet, so the section here, not the container :wink: